mic/rt_util.py

```diff
@@ -30,7 +31,7 @@
 BOOTSTRAP_URL="http://download.tizen.org/tools/micbootstrap"
 
 def runmic_in_runtime(runmode, opts, ksfile, argv=None):
-    dist = misc.get_distro()[0]
+    dist = distro.name()
```

Comment:
This needs also to the rpm dependency to python2-distro-info? Otherwise it gives error "ImportError: No module named distro" right?

Reply:
Nm. the last diff didn't show on my browser at first :D

Reply:
"Requires: python-distro" is there but note that this change belongs to the other PR - see the very first comment.